helm/configs/config.yaml

```diff
@@ -0,0 +1,298 @@
+http:
+  # Public URL base used when building absolute public URLs
+  public_base: {{ .Values.config.http.base | quote }}
+
+  # OIDC issuer advertised by the service. Defaults to `public_base`
+  issuer: {{ .Values.config.http.issuer | quote }}
+
+  # Each listener can serve multiple resources, and listen on multiple TCP ports or UNIX sockets.
+  # List of HTTP listeners, see below
+  listeners:
+    # The name of the listener, used in logs and metrics
+    - name: web
+
+      # List of resources to serve
+      resources: {{- range .Values.config.listeners.resources }}
+      - name: {{ .name }}
+      {{- if .path }}
+        path: {{ .path -}}
+      {{- end }}
+      {{- if .playground }}
+        playground: {{ .playground }}
+      {{- end }}
+      {{- end }}
```

Comment:
If anyway the chart exposes everything on a single port (which is fine!), I wouldn't expose the various resources in the values, but rather flags like `graphqlPlayground`, `metrics`, etc. and change that section accordingly

Reply:
which listener make sense to expose by default and which should always be optional besides `graphqlPlayground` and `metrics` ?